docs/en/observability/whats-new.asciidoc

```diff
@@ -3,15 +3,109 @@
+{observability-guide-all}/8.3/whats-new.html[8.3] |
+{observability-guide-all}/8.2/whats-new.html[8.2] |
+{observability-guide-all}/8.1/whats-new.html[8.1] |
+{observability-guide-all}/8.0/whats-new.html[8.0] |
+{observability-guide-all}/7.17/whats-new.html[7.17] |
 
 // tag::whats-new[]
 
-// What's new content goes in here. Don't uncomment or remove the tags surrounding this content :)
+[discrete]
+== Perform annotations-based monitoring of Kubernetes workloads with standalone Elastic Agent
+
+If you're using Elastic Agent in standalone mode, you can now perform annotation
+or _hints-based_ monitoring of Kubernetes workloads. This feature (already
+ available for Beats users) is now available as a technical preview in 8.5 for
+Elastic Agent running in standalone mode.
+
+This feature enables Elastic operators to provide Kubernetes cluster users with
+a self-service monitoring configuration experience that is GitOps friendly.
+You can enable this feature in the Elastic Agent YAML file by uncommenting
+the init container as shown:
+
+[role="screenshot"]
+image::images/hints-autodiscovery-yaml.png[Uncomment to enable hints-based monitoring]
+
+You can download Elastic Agent templates for https://github.com/elastic/elastic-agent/tree/main/deploy/kubernetes/elastic-agent-standalone/templates.d[~60 supported packages]
+, and add or edit your own templates within `/etc/elastic-agent/inputs.d`.
+
+Cluster users responsible for defining a pod manifest can add annotations that
+indicate that the pod contains a specific package they would like to monitor.
+Even if cluster users do not have access to the namespace where Elastic Agent is
+deployed, they are able to choose and configure monitoring for a specific pod.
+The following example configuration will enable Elastic Agent to monitor this
+pod using the configuration details for redis without further user action being
+required.
+
+[role="screenshot"]
+image::images/hints-config-example.png[Example annotations configuration for redis]
+
+For more info, https://www.elastic.co/guide/en/fleet/master/elastic-agent-kubernetes-autodiscovery.html#_hints_annotations_based_autodiscover[read the docs].
+
+[discrete]
+== New Istio integration added
+
+With a new Istio integration available in beta for version 8.5, you can collect
+logs from the Istio service mesh with Elastic Agent.
+
+The Istio service mesh extends Kubernetes to establish a programmable
+application-aware network using the powerful Envoy service proxy. Working with
+both Kubernetes and traditional workloads, Istio brings universal traffic
+management, telemetry, and security, to complex deployments. {integration-docs}/istio[Read the docs] for
+more information, and look out for support for metrics coming in future releases.
+
+[role="screenshot"]
+image::images/istio-integration.png[Add the Istio integration to collect logs from the Istio service mesh]
+
+[discrete]
+== View contextual Kubernetes info from APM Service Overview
```

Comment:
```suggestion
== View contextual Kubernetes info in the APM Service Overview
```
"from" makers it sound like the info is coming from the overview, not being viewed through it. I'm not sure you even need "in the APM Service Overview" here